.github/workflows/src/arm-incremental-typespec.js

```diff
@@ -92,7 +92,15 @@ export default async function incrementalTypeSpec({ core }) {
     }
 
     // If a readme is changed, to be conservative, handle as if every input file in the readme were changed
```

Comment:
no need to add a new test, since existing tests validate the new code matches the previous code